tests/test_flask.py

```diff
@@ -26,16 +32,11 @@ def test_markers_empty(app):
     #print(rv.data)
 
 
-# this was the request I saw when I loaded the main page
-def test_markers(app):
-    rv = app.get("/markers?ne_lat=32.08656790211843&ne_lng=34.80611543655391&sw_lat=32.08003198103277&sw_lng=34.793884563446&zoom=17&thin_markers=false&start_date=1104537600&end_date=1484697600&show_fatal=1&show_severe=1&show_light=1&approx=1&accurate=1&show_markers=1&show_discussions=1&show_urban=3&show_intersection=3&show_lane=3&show_day=7&show_holiday=0&show_time=24&start_time=25&end_time=25&weather=0&road=0&separation=0&surface=0&acctype=0&controlmeasure=0&district=0&case_type=0")
-    assert rv.status == '200 OK'
-    assert rv.headers['Content-Type'] == 'application/json'
-    #print(rv.data)
-    resp = json.loads(rv.data)
-    assert 'markers' in resp
-    assert len(resp['markers']) == 41
-    #print(len(resp['markers']))
+@pytest.fixture(scope="module")
+def marker_counter():
+    counter = Counter()
+    yield counter
+    assert counter['markers'] == 1688
```

Comment:
Is this for the total number of markers from all tests, or from a specific test?

Reply:
All tests. At first I wanted to hold a number for every test, but that's too much work.